packages/surveycto/src/Utils.js

```diff
@@ -0,0 +1,59 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+  makeBasicAuthHeader,
+} from '@openfn/language-common/util';
+
+const addBasicAuth = (configuration = {}, headers) => {
```

Comment:
I am seeing this get copied into a lot of adaptors. Maybe we need to just push it into common?

This isn't a job for now, just an observation